src/scenes/Question/scenes/Answer/components/Tips/Tips.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { default as TipsComponent } from 'components/Tips'
+
+const Tips = props => (
+  <TipsComponent {...props} uid="tips_answer">
+    <h3>Tips to write great answers on the FAQ</h3>
+    <div style={{ textAlign: 'center' }}>
+      <ul style={{ textAlign: 'left' }}>
+        <li>A partial answer is better than no answer.</li>
+        <li>Be precise and factual.</li>
+        <li>
+          A satisfactory answer must also be pleasant to read, formate it!
```

Comment:
That's a bit harsh ("must", "satisfactory", exclamation mark). I think maybe this would better communicate the intent that formatting is preferred but not required: "The best answers are pleasant to read and well formatted.". Let's not forget a significant portion of users don't know what markdown is.